test/get_test.js

```diff
@@ -3,7 +3,7 @@ goog.setTestOnly();
 
 const {DataLayerHelper} = goog.require('helper');
 
-describe('The helper.get function', function() {
+describe(`The 'get' function of helper`, () => {
```

Comment:
nit: I'd prefer to use single quotes for the full description, and ticks for the names of functions/methods within the description. Same everywhere else.